include/boost/gil/image_processing/hessian.hpp

```diff
@@ -0,0 +1,74 @@
+#ifndef BOOST_GIL_IMAGE_PROCESSING_HESSIAN_HPP
+#define BOOST_GIL_IMAGE_PROCESSING_HESSIAN_HPP
+
+#include <boost/gil/image_view.hpp>
+#include <boost/gil/typedefs.hpp>
+#include <stdexcept>
+
+namespace boost { namespace gil {
+
+/// \brief Computes Hessian response
+///
+/// Computes Hessian response based on computed entries of Hessian matrix, e.g. second order
+/// derivates in x and y, and derivatives in both x, y.
+/// Weights change perception of surroinding pixels.
+/// Additional filtering is strongly advised.
+template <typename GradientView, typename Weights, typename OutputView>
+inline void compute_hessian_responses(
+    GradientView ddxx,
```

Comment:
`ddxx`, `dxdy` and `ddyy` are pretty cryptic. Please, either consider more descriptive names or `/// \param ...` describe them.

Reply:
I adjusted docs to include an example of the notation, hopefully it will be more clear now. Could you please resolve this and the one below if it gets better?